bin/applypatch

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
If these scripts are openssl specific can we put them all under another directory?

Reply:
I thought they might end up being useful for other projects but I do need to clean them up to make them general purpose.

Reply:
You are right though - for now let me move them out of the way.

Reply:
i moved the code and cleaned it up and generalized it. Please re-review @fanbo-meng

Reply:
i am marking this as resolved since @fanbo-meng is out. @fanbo-meng please open an issue if you still see problems.